packages/purpose-process/test/integration/createReversePurpose.test.ts

```diff
@@ -348,12 +356,16 @@ describe("createReversePurpose", () => {
     await addOneTenant(producerDelegate);
     await addOneAgreement(mockAgreement);
 
-    const { purpose, isRiskAnalysisValid } =
+    const createReversePurposeResponse =
```

Comment:
We have an expect only on the written payload, but not on the return value of the service call, I think. I would add also and expect like this below:

`expect(createReversePurposeResponse).toEqual({
   purpose: expectedPurpose,
   isRiskAnalysisValid: expectedIsRiskAnalysisValid
})`